### PR TITLE
fix deployment, solc / truffle config, specify ABI / contract address…

### PR DESCRIPTION
**Pull request by rootslashbin**
…, openzeppelin imports, package additions / updates, etc

### CODE REVIEW

File: src/providers/MoralisDappProvider/MoralisDappProvider.js [hunk 1]
```diff
@@ -5,9 +5,9 @@ import MoralisDappContext from "./context";
...
+  const [marketAddress, setMarketAddress] = useState('0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1'); //Smart Contract Address Here
```
Comment (mansamusai) [inline src/providers/MoralisDappProvider/MoralisDappProvider.js:10]: is this just a dummy addr?

Comment (mansamusai) [inline src/providers/MoralisDappProvider/MoralisDappProvider.js:10]: small nitpick but maybe better to have a separate file for the ABI

**Review by mansamusai — APPROVED** [review]
lgtm man. thanks for contributing!

File: src/providers/MoralisDappProvider/MoralisDappProvider.js (hunk 1 above)
Comment (rootslashbin) [reply src/providers/MoralisDappProvider/MoralisDappProvider.js:10]: yeah, that's the address it got deployed to on my local ganache instance.

Comment (rootslashbin) [reply src/providers/MoralisDappProvider/MoralisDappProvider.js:10]: probably is, but JS isn't my primary language :<

wholeheartedly welcome any and all improvements <3

**Pull request merged**